banking/README.md

```diff
@@ -0,0 +1,151 @@
+# Java Cloud Spanner Sample Online Banking Application
+
+A sample online banking application that uses the Cloud Spanner advanced features, including [transactions](https://cloud.google.com/spanner/docs/transactions), [AI integration](https://cloud.google.com/spanner/docs/ml), [full-text search](https://cloud.google.com/spanner/docs/full-text-search), and [BigQuery federated queries](https://cloud.google.com/spanner/docs/databoost/databoost-run-queries).
+
+This sample requires [Java](https://www.java.com/en/download/) and [Maven](http://maven.apache.org/) for building the application.
+
+The accompanying codelab for this sample is: [https://codelabs.developers.google.com/spanner-online-banking-app](https://codelabs.developers.google.com/spanner-online-banking-app)
+
+## Build and Run
+
+ℹ️ Note that you can use [Google Cloud Shell](https://cloud.google.com/shell/docs) instead of developing locally.  In that case skip step #2 and #3 below.
+
+1. Create a project in the [Google Cloud Platform console](https://console.cloud.google.com/).
+
+2. Install and configure the [gcloud CLI](https://cloud.google.com/sdk/docs/install-sdk).
+
+3. Follow the [Java development environment setup instructions](https://cloud.google.com/java/docs/setup).
```

Comment:
It's part of this step, but might call it out separately since authentication is so important and might be missed. 

gcloud auth application-default login